Update README.md (#6)
searchableguy/whiz: README.md

wz list recent github branches sorted by activity
```

## Privcacy

> This script will send the following information to ChatGPT:

Reviewer: ```suggestion
Whiz will send the following information to OpenAI:
```

- Your `process.platform`, one of ‘aix’, ‘android’, ‘darwin’, ‘freebsd’, ‘linux’, ‘openbsd’, ‘sunprocess’, or ‘win32’
- Your `process.arch`, one of  ‘x32’, ‘x64’, ‘arm’, ‘arm64’, ‘s390’, ‘s390x’, ‘mipsel’, ‘ia32’, ‘mips’, ‘ppc’, or ‘ppc64’
- The basename of your shell path: `process.env.SHELL?.split("/").pop()`, whatever that is

> - Your input prompt

Reviewer: ```suggestion
- Your request
```

> It will not send anything else about your environment like filenames or file content to ChatGPT.

Reviewer: ```suggestion
No other information is shared with OpenAI.
```
